pyccel/cuda.py

```diff
@@ -0,0 +1,13 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+This module contains CUDA methods for Pyccel.
```

Comment:
I am not convinced that we will be able to (or should) describe all methods that may be required for cuda in one file without submodules. I would prefer creating a module for this and using an `__init__.py` file to keep the same import syntax. Otherwise changing to a module setup later may be a breaking change

Reply:
I agree that's gonna make the code more organized I will see what I can do

Reply:
can u check now please [ba56731](https://github.com/pyccel/pyccel-cuda/pull/42/commits/ba56731764dc5314c04d11dc33ff9c9bafc2dcb7)

Reply:
can u check now https://github.com/pyccel/pyccel-cuda/commit/4307d679d1682b0e433f20c84b42e0f3e612c2d7

Reply:
I think this is better